ethereum/networks/src/test/java/tech/pegasys/teku/networks/EphemeryNetworkTest.java

```diff
@@ -0,0 +1,247 @@
+  private SpecConfig configFile;
+  private SpecConfig config;
+
+  @BeforeEach
+  void setUp() {
+    periodSinceGenesis = EphemeryNetwork.getPeriodsSinceGenesis(timeProvider);
+    expectedMinGenesisTime = MIN_GENESIS_TIME + (periodSinceGenesis * PERIOD_IN_SECONDS);
+    expectedChainId = GENESIS_CHAINID + periodSinceGenesis;
+    builder = mock(SpecConfigBuilder.class);
+    configFile = SpecConfigLoader.loadConfig("ephemery");
+    config = mock(SpecConfig.class);
+  }
+
+  @Test
+  public void testUpdateConfig() {
+    when(config.getRawConfig()).thenReturn(configFile.getRawConfig());
+    when(builder.rawConfig(config.getRawConfig())).thenReturn(builder);
+    when(builder.depositChainId(expectedChainId)).thenReturn(builder);
+    when(builder.depositNetworkId(expectedChainId)).thenReturn(builder);
+    when(builder.minGenesisTime(UInt64.valueOf(expectedMinGenesisTime))).thenReturn(builder);
+
+    EphemeryNetwork.updateConfig(builder);
+
+    verify(builder, times(1)).rawConfig(config.getRawConfig()); // Only verify once
+    verify(builder, times(1)).depositNetworkId(expectedChainId);
+    verify(builder, times(1)).depositChainId(expectedChainId);
+    verify(builder, times(1)).minGenesisTime(UInt64.valueOf(expectedMinGenesisTime));
+    assertThat(CURRENT_TIMESTAMP).isGreaterThan(GENESIS_CHAINID + PERIOD_IN_SECONDS);
+  }
+
+  @Test
+  public void shouldLoadDepositNetworkId() {
+    final Spec spec = getSpec(phase0Builder -> phase0Builder.depositNetworkId(GENESIS_CHAINID));
+
+    assertThat(spec.getGenesisSpec().getConfig().getRawConfig().get("DEPOSIT_NETWORK_ID"))
+        .isEqualTo(String.valueOf(GENESIS_CHAINID));
+  }
+
+  @Test
+  public void shouldLoadDepositChainId() {
+    final Spec spec = getSpec(phase0Builder -> phase0Builder.depositChainId(GENESIS_CHAINID));
+
+    assertThat(spec.getGenesisSpec().getConfig().getRawConfig().get("DEPOSIT_CHAIN_ID"))
+        .isEqualTo(String.valueOf(GENESIS_CHAINID));
+  }
+
+  @Test
+  public void shouldLoadMinGenesisTime() {
+    final Spec spec =
+        getSpec(phase0Builder -> phase0Builder.minGenesisTime(UInt64.valueOf(MIN_GENESIS_TIME)));
+
+    assertThat(spec.getGenesisSpec().getConfig().getRawConfig().get("MIN_GENESIS_TIME"))
+        .isEqualTo(String.valueOf(MIN_GENESIS_TIME));
+  }
+
+  @Test
+  public void read_missingConfig() {
+    processFileAsInputStream(getInvalidConfigPath("missingChurnLimit"), this::readConfig);
+
+    assertThatThrownBy(reader::build)
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessageContaining("MIN_PER_EPOCH_CHURN_LIMIT");
+  }
+
+  @Test
+  void shouldNotUpdateConfigBeforeGenesis() {
+    final Spec spec =
+        getSpec(phase0Builder -> phase0Builder.minGenesisTime(UInt64.valueOf(MIN_GENESIS_TIME)));
+
+    final long preGenesisTime = MIN_GENESIS_TIME - ONE_PERIOD;
+    final StubTimeProvider stubTimeProvider =
+        StubTimeProvider.withTimeInMillis(preGenesisTime * 1000);
```

Comment:
nit: could use `withTimeInSeconds` if you don't want to do the multiplication everywhere...

Reply:
Okay noted. Do I need to make the change for this one?